src/ontology/edits/oeo-model.omn

```diff
@@ -470,1114 +470,1424 @@ Class: <http://purl.obolibrary.org/obo/IAO_0000300>
-        DataFormat
+        OEO_00000120
     
     
-Individual: Coin-Or_CBC
+Individual: OEO_00000130
 
+    Annotations: 
+        rdfs:label "dict"
+    
     Types: 
-        Solver
+        OEO_00000120
     
     
-Individual: DataFrame
+Individual: OEO_00000134
 
+    Annotations: 
+        rdfs:label "dynamic programming"
+    
     Types: 
-        DataFormat
+        OEO_00000265
     
     
-Individual: Dict
+Individual: OEO_00000136
 
+    Annotations: 
+        rdfs:label "econometric"
+    
     Types: 
-        DataFormat
+        OEO_00000270
     
     
-Individual: DynamicProgramming
+Individual: OEO_00000138
 
+    Annotations: 
+        rdfs:label "economic equilibrium"
+    
     Types: 
-        MathematicalConcept
+        OEO_00000270
     
     
-Individual: Econometric
+Individual: OEO_00000163
 
+    Annotations: 
+        rdfs:label "false"
+    
     Types: 
-        MethodicalFocus
+        OEO_00000078
     
     
-Individual: EconomicEquilibrium
+Individual: OEO_00000170
 
+    Annotations: 
+        rdfs:label "fortran"
+    
     Types: 
-        MethodicalFocus
+        OEO_00000382
     
     
-Individual: False
+Individual: OEO_00000179
 
+    Annotations: 
+        rdfs:label "fuzzy logic"
+    
     Types: 
-        BooleanVariable
+        OEO_00000265
     
     
-Individual: Fortran
+Individual: OEO_00000180
 
+    Annotations: 
+        rdfs:label "gams"
+    
     Types: 
-        SoftwareFramework
+        OEO_00000382
     
     
-Individual: FuzzyLogic
+Individual: OEO_00000187
 
+    Annotations: 
+        rdfs:label "gdx"
+    
     Types: 
-        MathematicalConcept
+        OEO_00000120
     
     
-Individual: GAMS
+Individual: OEO_00000195
 
+    Annotations: 
+        rdfs:label "glpk"
+    
     Types: 
-        SoftwareFramework
+        OEO_00000392
     
     
-Individual: GLPK
+Individual: OEO_00000196
 
+    Annotations: 
+        rdfs:label "gnu"
+    
     Types: 
-        Solver
+        OEO_00000382
     
     
-Individual: GNU
+Individual: OEO_00000203
 
+    Annotations: 
+        rdfs:label "gurobi"
+    
     Types: 
-        SoftwareFramework
+        OEO_00000392
     
     
-Individual: Gurobi
+Individual: OEO_00000215
 
+    Annotations: 
+        rdfs:label "hybrid"
+    
     Types: 
-        Solver
+        OEO_00000057
     
     
-Individual: Hybrid
+Individual: OEO_00000225
 
+    Annotations: 
+        rdfs:label "inactive"
+    
     Types: 
-        AnalyticalApproach
+        OEO_00000281
     
     
-Individual: Inactive
+Individual: OEO_00000244
 
+    Annotations: 
+        rdfs:label "java"
+    
     Types: 
-        Modus
+        OEO_00000382
     
     
-Individual: Java
+Individual: OEO_00000254
 
+    Annotations: 
+        rdfs:label "linear programming"
+    
     Types: 
-        SoftwareFramework
+        OEO_00000265
     
     
-Individual: LinearProgramming
+Individual: OEO_00000262
 
+    Annotations: 
+        rdfs:label "macro- economic"
```

Comment:
```suggestion
        rdfs:label "macro-economic"
```

Reply:
```suggestion
        rdfs:label "macroeconomic"
```
[Oxford](https://www.lexico.com/definition/macroeconomic) and [Merriam-Webster](https://www.merriam-webster.com/dictionary/macroeconomic) say so. ;)

Reply:
@0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q I implemented it your way now